plugins/kotlin/j2k/k1.new.post-processing/src/org/jetbrains/kotlin/idea/j2k/post/processing/inference/nullability/NullabilityDefaultStateProvider.kt

```diff
@@ -6,6 +6,7 @@ import org.jetbrains.kotlin.descriptors.TypeParameterDescriptor
 import org.jetbrains.kotlin.idea.j2k.post.processing.inference.common.*
 import org.jetbrains.kotlin.idea.refactoring.isAbstract
 import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.nj2k.tree.JKArrayGetterExpression
```

Comment:
I suggest to ignore all nullability issues for now, because it may be challenging to fix the current nullability analysis to recognize array access expressions properly.